### PR TITLE
Added gif support for pat :)

### PR DESCRIPTION
**Pull request by prono69**
Me nub. All thanks to Meli Odas :)

### CODE REVIEW

**Comment by Krishna-Singhal** [comment]
@prono69 Fix deepsource too

**Comment by prono69** [comment]
@Krishna-Singhal sir wo problem pehle se tha. Maine usko edit hi nehi kiya . :(

_Sent from my Redmi Note 4 using [FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)_

**Comment by prono69** [comment]
Also I don't know how to fix antipattern :(

File: plugins/pat.py [hunk 1]
```diff
@@ -12,23 +13,36 @@
...
-        await message.edit("**Bruh** ~`Reply to a message or provide username`")
+    reply = message.reply_to_message
+    reply_id = reply.message_id if reply else None
+    if not username and not reply:
+        await message.edit("**Bruh** ~`Reply to a message or provide username`", del_in=3)
         return
 
     resp = requests.get("http://headp.at/js/pats.json")
     pats = resp.json()
     pat = BASE_URL.format(parse.quote(choice(pats)))
```
Comment (Krishna-Singhal) [inline plugins/pat.py:27]: Change this Variable name

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

File: plugins/pat.py (hunk 1 above)
Comment (Krishna-Singhal) [reply plugins/pat.py:27]: change pat to _pat

**Comment by prono69** [comment]
Done sir . Thanks :)

**Comment by Jigarvarma2005** [comment]
Add [this](https://github.com/UsergeTeam/Userge-Plugins/blob/123223ac075602bc75b50fc9364621998d19121c/plugins/pat.py#L25#L29) lines after  [this lines](https://github.com/UsergeTeam/Userge-Plugins/blob/123223ac075602bc75b50fc9364621998d19121c/plugins/pat.py#L38)

**Comment by prono69** [comment]
@Jigarvarma2005 not working sir ? Or what ?

_Sent from my Redmi Note 4 using [FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)_

**Comment by Jigarvarma2005** [comment]
> @Jigarvarma2005 not working sir ? Or what ?
> 
> _Sent from my Redmi Note 4 using [FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)_

line 25-29 will be executed for nothing that will only use the cpu if -g flag

**Comment by Jigarvarma2005** [comment]
So do https://github.com/UsergeTeam/Userge-Plugins/pull/196#issuecomment-821799428 this

**Comment by Jigarvarma2005** [comment]
Now ok

**Comment by prono69** [comment]
OwO . Thanks

**Pull request merged**